lib/internal/cli_table.js

```diff
@@ -65,18 +71,21 @@ const table = (head, columns) => {
     }
   }
 
-  const divider = columnWidths.map((i) =>
-    tableChars.middleMiddle.repeat(i + 2));
+  const divider = ArrayPrototypeMap(columnWidths, (i) =>
+    StringPrototypeRepeat(tableChars.middleMiddle, i + 2));
 
-  let result = `${tableChars.topLeft}${divider.join(tableChars.topMiddle)}` +
+  let result = `${tableChars.topLeft}` +
```

Comment:
Should this be changed to this too?
(couldn't `suggest changes` because of the intermittent deleted lines)
```js
  let result = tableChars.topLeft +
               ArrayPrototypeJoin(divider, tableChars.topMiddle) +
               tableChars.topRight + '\n' +
               renderRow(head, columnWidths) + '\n' +
               tableChars.leftMiddle +
               ArrayPrototypeJoin(divider, tableChars.rowMiddle) +
               tableChars.rightMiddle + '\n';
```